.github/workflows/update-miner.yml

```diff
@@ -1,9 +1,9 @@
-name: Update Miner GA (Testnet)
+name: Update Gatewayrs GA (Testnet)
 on:
   repository_dispatch:
-    types: [miner-update]
+    types: [gatewayrs-update]
 jobs:
-  miner-update:
+  gatewayrs-update:
```

Comment:
Until we land the light-miner work, we don't want to overwrite the existing testnet GA bumping behavior do we?

Reply:
by mistake raised against master. The PR should have been against light-hotspot branch.